### ISSUE TITLE
NVDA thinks there's an extra item when reporting position information in certain programs

### ISSUE THREAD

**Issue by SeedyThreeSixty** [nvaccess/nvda#15067]
### Steps to reproduce:
Before performing these steps, ensure that the Guess object position information when unavailable option is enabled in NVDA's object presentation settings (NVDA Control O).
#### in Foobar2000

* In Windows Explorer, navigate to a folder with some audio files in it. It doesn't matter what type of audio files they are, so long as they're an audio files.
* Select all the files with Control A, Control C to copy the files, open Foobar2000, tab to the list view (you might have to do this multiple times depending on how you have Foobar2000 configured), then paste the files with Control V.

#### In WinSCP

* Launch WinSCP and log into your server.
* Tab to the list view and navigate to a directory within the server.
### Actual behavior:

NVDA will think there is an extra item in the list when there actually isn't. Take this Foobar2000 example:
KPM Music - KPM - FTV Online (APM KPM FTVOL) - MIXED BAG 4 (FTVOL-0004)  1.49 Analyse // APM Music 3:00  2 of 13
As you can see, NVDA said 2 of 13, but there are only 12 items in the list of files.
### Expected behavior:

What NVDA should have said was:
KPM Music - KPM - FTV Online (APM KPM FTVOL) - MIXED BAG 4 (FTVOL-0004)  1.49 Analyse // APM Music 3:00  2 of 12
There are 12 items in the list, so the position of that item should be 2 of 12, not 2 of 13.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.1
#### Windows version:
Windows 8.1 (AMD64) build 9600.20778
#### Name and version of other software in use when reproducing the issue:
* Foobar2000
* WinSCP
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The first version of NVDA I remember using when I first found this weird bug was NVDA 2022.2. The bug has been in every version since then.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by michaelDCurran** [comment]
Per the user guide entry for the Guess Object position information setting, it may be inaccurate in some cases. Unfortunately there is no way we can fix this in all scenarios. We would generally recommend that this setting be left switched off, unless you are happy to accept possible inaccurate information.
